sd-card/html/ota_page.html

```diff
@@ -168,6 +168,8 @@ <h2>Reboot</h2>
         alert("No file selected!");
     } else if (filePath.length == 0) {
         alert("File path on server is not set!");
+    } else if (filePath.length > 30) {
+        alert("Filename is to long! Use a shorter name.");
```

Comment:
@haverland Please provide the number, else people have to do try&error until it fits the check

Reply:
added